tools/mo/openvino/tools/mo/main.py

```diff
@@ -32,29 +32,20 @@ def main(cli_parser: argparse.ArgumentParser, framework=None):
     # Initialize logger with 'ERROR' as default level to be able to form nice messages
     # before arg parser deliver log_level requested by user
     init_logger('ERROR', False)
-    logger = log.getLogger()
-    # Disable logging for parse_args() as inner convert runs parse_args() second time
-    # which result in duplicating of warnings
-    logger.disabled = True
-    argv = cli_parser.parse_args()
-    logger.disabled = False
-    argv.model_name = get_model_name_from_args(argv)
-    is_tf, _, _, _, _ = deduce_legacy_frontend_by_namespace(argv)
-    argv = vars(argv)
-
-    if framework is not None:
-        argv['framework'] = framework
 
     ngraph_function = None
+    argv = None
+    is_tf = False
     try:
-        ngraph_function = convert_model(**argv)
+        ngraph_function, argv = _convert(cli_parser, framework, {})
+        is_tf, _, _, _, _ = deduce_legacy_frontend_by_namespace(argv)
```

Comment:
please swap these two lines because in case of failure of conversion of TF model using TF FE `is_tf` is False so we cannot raise info message. Please check it.

Reply:
But we don't have argv until we run `_convert()`. To obtain args from cli_parser we need to run `parse_args()` which is moved to `_convert()`.

Reply:
By the way, do we still need to print get_tf_fe_legacy_message() if we plan to switch to TF FE by default?

Reply:
ok, agreed. I will remove it in my PR for switching to TF FE.